ibm_catalog.json

```diff
@@ -84,6 +78,12 @@
                 "crn:v1:bluemix:public:iam::::role:Editor"
               ],
               "service_name": "cloud-object-storage"
+            },
+            {
+              "role_crns": [
+                "crn:v1:bluemix:public:iam::::role:Administrator"
+              ],
+              "service_name": "all-account-management-services (Required if user is not the Cloud accout owner)"
```

Comment:
This is not a valid value. I tried to see what the json picked up in the UI, and it returned below. I don't think it is possible to add this type of access to the catalog manifest. It only support passing a service name (not a service type) 
![image](https://github.com/user-attachments/assets/33761710-8e51-49fd-85af-9cf5641f0551)

Reply:
Looks like catalog does not support but it does not validate too. So currently possible and no error. Anyways after our conversation, I have removed it now.